test/e2e/suite/smoke-tests/addons_test.go

```diff
@@ -0,0 +1,120 @@
+package smoketest
```

Comment:
let's name this package `addon` or `addons` if you expect a lot of name collisions with an `addon` variable